fudge-ratio/src/main/webapp/script.js

```diff
@@ -6,6 +6,15 @@ let totalMinutes = 0;
 let hours = 0;
 let timer;
 
+// Checks when the start and stop buttons are clicked once the HTML page is loaded.
+window.addEventListener("load", activateListeners);
```

Comment:
nice, really considerate for you to check if we have actually load all the DOM elements before you tried to access them, another idea I had was using https://twitter.com/slicknet/status/1238915727416950784
but this is totally valid :D